SUPPORTING_NEW_SERVICES.md

```diff
@@ -0,0 +1,108 @@
+* Create a README in the service's folder.  This README should mimic the structure of other services' READMEs.  In particular, you should create a step-by-step "Getting Started" guide.  See [`gcloud-java-datastore`'s README](https://github.com/GoogleCloudPlatform/gcloud-java/blob/master/gcloud-java-datastore/README.md) for reference.  All code in that step-by-step guide should also be included in the `gcloud-java-examples` snippets package.
+* The API and test helper packages should have `package-info.java` files.  These files should contain descriptions of the packages as well as simple example code and/or links to code snippets.
+* Public methods, classes, and builders should contain meaningful Javadoc.  When possible, copy docs from the service's `cloud.google.com` API documentation and provide `@see` links to relevant Google Cloud web pages.  Document both unchecked and checked exceptions.
+* Update [`TESTING`](https://github.com/GoogleCloudPlatform/gcloud-java/blob/master/TESTING.md) with how to run tests using the test helper.
+* Update the [`gcloud-java-examples` README](https://github.com/GoogleCloudPlatform/gcloud-java/blob/master/gcloud-java-examples/README.md) with instructions on how to run your example application.
+
+Notes/reminders:
+* Clearly document which APIs must be enabled in the Developers Console's API Manager.
+* Versioning in documentation is automatically updated by the script `utilities/update_docs_version.sh`.  Be sure to examine that script to make sure any version-dependent documentation will be updated properly upon release.
+
+### Workflow
+
+New services should be created in a branch based on `master`.  The branch name should include the suffix "-alpha".  For example, while developing `gcloud-java-pubsub`, all Pub/Sub related work should be done in `pubsub-alpha`.  All code should be submitted through pull requests from a branch on a forked repository.  Limiting pull request size is very helpful for reviewers.  All code that is merged into the branch should be standalone and well-tested.  Any todo comments in the code should have an associated Github issue number for tracking purposes.  You should periodically pull updates from the master branch, especially if there are project-wide updates or if relevant changes have been made to the core utilities library, `gcloud-java-core`.
+
+Create at least two milestones (stable and future) for your service and an issue tag with the service name.  Create issues for any to-do items and tag them appropriately.  This keeps an up-to-date short-term to-do list and also allows for longer term roadmaps.
+
+Be sure you've configured the base folder's `pom.xml` correctly.
+  * Add your module to the base directory's `pom.xml` file under the list of modules (in alphabetical order).
+  * Add your module to the javadoc packaging settings.  See PR #802 for an example.
```

Comment:
Oh, ignore my comment above regarding spi and the note to update the main pom as I see that you are mentioning it here.